cg/cli/workflow/balsamic/base.py

```diff
@@ -169,9 +170,7 @@ def store_housekeeper(
         analysis_api.verify_case_config_file_exists(case_id=case_id, dry_run=dry_run)
         analysis_api.verify_deliverables_file_exists(case_id=case_id)
         analysis_api.upload_bundle_housekeeper(case_id=case_id, dry_run=dry_run, force=force)
-        analysis_api.upload_bundle_statusdb(
-            case_id=case_id, comment=comment, dry_run=dry_run, force=force
-        )
+        analysis_api.update_analysis_statusdb(case_id=case_id, comment=comment, dry_run=dry_run)
```

Comment:
functionality for the `force` flag is not needed anymore as it just skipped updating the `completed_at` column in Analysis, a feature that we want, according to production (double check with KN)